File: my-node-app/learning/旧/5. 模拟请求.js
let url = 'https://www.mntuce01.com/wp-admin/admin-ajax.php';
let postId = 19431;
let nextPage = 2;

const response = await fetch(url, {
  method: 'POST',
  // headers: {
  //   'Content-Type': 'application/json',
  // },
  body: JSON.stringify({
    action: 'load_more_post_page',
    post_id: postId,
    page_num: nextPage
  })
});

const data = await response.json();
console.log(data);

File: my-node-app/learning/旧/5. 模拟请求.py
import requests

# url = 'https://www.mntuce01.com/wp-admin/admin-ajax.php'
# postId = 19431
# nextPage = 2
#
# data = {
#     'action': 'load_more_post_page',
#     'post_id': postId,
#     'page_num': nextPage
# }
#
#
# response = requests.post(url, data=data, verify=False)
# print(response.json())

start_url = "https://www.mntuce01.com/19357/.html"

r = requests.get(start_url)
print(r.text)